Girraj/ss.java

```diff
@@ -0,0 +1,97 @@
+package javaapplication1;

```

Comment:
package name should be seperated by dots. 
It should be name java.application.1
Actually should be more specific, java.application.1 does not tell whats inside this package.
Check at the package structure, pallav created.